plugins/stack-overflow/src/home/StackOverflowQuestions/Content.tsx

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright 2022 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { useApi, configApiRef } from '@backstage/core-plugin-api';
+import { Link } from '@backstage/core-components';
+import {
+  IconButton,
+  List,
+  ListItem,
+  ListItemText,
+  ListItemSecondaryAction,
+} from '@material-ui/core';
+import OpenInNewIcon from '@material-ui/icons/OpenInNew';
+import { useAsync } from 'react-use';
+import qs from 'qs';
+import React from 'react';
+import {
+  StackOverflowQuestion,
+  StackOverflowQuestionsContentProps,
+} from '../../types';
+
+/**
+ * A component to display a list of stack overflow questions on the homepage.
+ *
+ * @public
```

Comment:
I noticed at this point - this doesn't have an api report does it? I think as you run `yarn build:api-reports` you'll run into some of the issues I've mentioned elsewhere (tsc will give some warnings etc).

But what got me asking initially is, I wonder if this really is part of the public surface, and if it is, should it be renamed?

Reply:
oh yeah good point, this should be enough to have when the component is exported as a homepage component. Thanks for highlighting!